src/endpoint.ts

```diff
@@ -184,7 +184,7 @@ export class Endpoint<
           })),
         ]);
       }
-      throw e;
+      throw e instanceof Error ? e : new Error(`${e}`);
```

Comment:
i literally just wrote this exact line of code in my own repo to coerce prettier's fucky string errors LMAO

Reply:
it turned out that `Symbol` require special handling